queries/php/highlights.scm

```diff
@@ -173,14 +214,17 @@
 [
   "="
 
+  "."
```

Comment:
Why is this an operator? In other languages foo.bar is punctuation. If this needs a change it should be applied to all languages

Reply:
Dot is the [string concatenation operator](https://www.php.net/manual/en/language.operators.string.php):
```php
<?php
$a = 'some ' . 'text';
```

In PHP object properties are always accessed with the arrow `->` not the `.`: 
```php
<?php
$a->someProp;
$a->someMethod();
```

Static props/methods use the `::` operator:
```php
<?php
$a = SomeClass::$prop;
$a = SomeClass::someMethod();
```

Reply:
Yeah, I had the suspicion that is was concatenation but didn't whether it was also object access